browser/js/bundle.entry.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  bel: require("bel"),
+  morphdom: require('morphdom'),
```

Comment:
Yea, I'm kinda using it both ways right now. I considered using yo-yo but I didn't really need anything else from it so I shied away.